### DISCUSSION TITLE
[General] <Title>Problem Discord

### DISCUSSION THREAD

**Discussion by StiMuL8066** [zkSync-Community-Hub/zksync-developers#332] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I can't log into discord

### What is the specific issue or error you're encountering?

unable to accept invitation

### Can you share the error messages or logs you're receiving, if any?

![PC](https://github.com/zkSync-Community-Hub/zksync-developers/assets/121286862/062376a4-7725-41f7-b622-f8a679bf300a)
![Android](https://github.com/zkSync-Community-Hub/zksync-developers/assets/121286862/a31d0745-8e98-4e28-b5ea-419c2330740d)

### Have you made any recent changes to the contract before encountering this issue?

No

### Are there any external libraries or contracts that your contract interacts with?

No

### Can you provide the relevant portions of your contract code where the issue is occurring?

No

### Have you tried to isolate the problem, and if so, what were the results?

I tried to log into discord from my phone and from my PC, different errors, screenshots above

### What steps have you already taken to try to resolve the issue?

I tried logging into discord from my phone and PC, using VPN

### Additional Details

For some reason I was kicked out of discord, my nickname is stimul8066. I am an active zksync user. At the moment I can’t log into discord, it gives an error, I’m attaching a photo. There are no advertisements in my nickname, nor in my biography at all, nor in my profile. There are different errors from the PC and from the phone."

**Comment by bxpana** [comment]
Please reach out to us at support@zksync.io. This is for Developer support
